lib/docker/executeDockerBuild.ts

```diff
@@ -256,47 +258,61 @@ async function dockerPush(images: string[],
                           options: DockerOptions,
                           gi: ProjectAwareGoalInvocation): Promise<ExecuteGoalResult> {
 
-    let result = Success;
-
     if (await pushEnabled(gi, options)) {
 
-        // 1. run docker push
-        for (const image of images) {
-            result = await gi.spawn(
-                "docker",
-                ["push", image],
-                {
-                    env: {
-                        ...process.env,
-                        DOCKER_CONFIG: dockerConfigPath(options, gi.goalEvent),
+        // Login to registry(s)
+        await Promise.all(
```

Comment:
Running concurrent login will ultimately create issues when all docker processes attempt to write to the same config file. Please change to sequential.

Reply:
👍